src/tooltips/rnaEditingTooltip.ts

```diff
@@ -0,0 +1,54 @@
+import { TransformedRnaEditing } from '../adapters/types/rna-editing';
+
+const getREDIportalId = (feature: TransformedRnaEditing) =>
+  feature.dbReferenceType.find((db) => db.type === 'rna_editing')?.id;
+
+const getREDIportalLink = (id: string) =>
+  `http://srv00.recas.ba.infn.it/cgi/atlas/getpage_dev.py?query9=hg&query10=hg38&acc=${id}`;
+
+const getEnsemblLink = (id: string) => `https://www.ensembl.org/id/${id}`;
+
+const getLinks = (feature: TransformedRnaEditing) => {
+  const links = [];
+  const rediPortalId = getREDIportalId(feature);
+  if (rediPortalId) {
+    links.push(
+      `REDIportal <a href="${getREDIportalLink(
+        rediPortalId
+      )}">${rediPortalId}</a>`
```

Comment:
```suggestion
      )}" target="_blank">${rediPortalId}</a>`
```
It will be better to have them in open tabs.